com.unity.multiplayer.mlapi/Runtime/Core/NetworkObject.cs

```diff
@@ -27,20 +26,94 @@ public sealed class NetworkObject : MonoBehaviour
 #if UNITY_EDITOR
         private void OnValidate()
```

Comment:
fixes (we figured today together) for `NetworkObject.GlobalObjectIdHash` should go (and will go) under a separate PR on its own :)

Reply:
Leaving the core fixes in until the other PR is merged, then we can merge that back into this PR and this PR should be good to go.